### ISSUE TITLE
Only train last gradients and loss

### ISSUE THREAD

**Issue by carpedm20** [carpedm20/NTM-tensorflow#1]
Branch [only_train_last_gradients](https://github.com/carpedm20/NTM-tensorflow/tree/only_train_last_gradients)

**Comment by carpedm20** [comment]
fixed
